src/InverseCdf.cpp

```diff
@@ -0,0 +1,47 @@
+#include "InverseCdf.h"
+#include <math.h>
```

Comment:
Tiny thing - we've shifted to using <cmath>

Reply:
Done.